src/styles/base/_colors.scss

```diff
@@ -125,25 +125,25 @@ $color-var: (
 );
 
 @each $name, $color in $color-var {
-  .border-#{$name} {
+  .border-#{"" + $name} {
```

Comment:
the new sass package throws a warning that these should be strings and this is how to resolve it.